### ISSUE TITLE
Training loss, training code

### ISSUE THREAD

**Issue by fftobiwan** [iorism/CNN#1]
Hi,

I haven't found an email address to directly contact you, hence I am writing on Github :)

I have a question regarding the training loss. Is it defined as the L2 loss on the wavelet coefficients or is it defined in image space (ie by reconstructing the clean image from the predicted wavelet coefficients and then calculating the squared error to the ground truth)?

Is it possible to provide code for training your network?

Thanks and best regards,
Tobias Plötz
